datetime/datetime.go

```diff
@@ -79,16 +77,44 @@ type Datetime struct {
 	time time.Time
 }
 
+const (
+	// NoTimezone allows to create a datetime without UTC timezone for
+	// Tarantool. The problem is that Golang by default creates a time value
+	// with UTC timezone. So it is a way to create a datetime without timezone.
+	NoTimezone = ""
+)
+
+var noTimezoneLoc = time.FixedZone(NoTimezone, 0)
+
+const (
+	offsetMin = -12 * 60 * 60
+	offsetMax = 14 * 60 * 60
+)
+
 // NewDatetime returns a pointer to a new datetime.Datetime that contains a
-// specified time.Time. It may returns an error if the Time value is out of
-// supported range: [-5879610-06-22T00:00Z .. 5879611-07-11T00:00Z]
+// specified time.Time. It may return an error if the Time value is out of
+// supported range: [-5879610-06-22T00:00Z .. 5879611-07-11T00:00Z] or
+// an invalid timezone or offset value is out of supported range:
+// [-12 * 60 * 60, 14 * 60 * 60].
 func NewDatetime(t time.Time) (*Datetime, error) {
 	seconds := t.Unix()
 
 	if seconds < minSeconds || seconds > maxSeconds {
 		return nil, fmt.Errorf("Time %s is out of supported range.", t)
 	}
 
+	zone, offset := t.Zone()
+	if zone != NoTimezone {
+		if _, ok := timezoneToIndex[zone]; !ok {
+			return nil, fmt.Errorf("Unknown timezone %s with offset %d",
+				zone, offset)
```

Comment:
Why do we use capitalized error strings? I have found some reasonable explanation why should not we do it. (See [this page](https://github.com/golang/go/wiki/CodeReviewComments#error-strings))

Reply:
If you are agree with that point, we would need some code health patch for fixing it.

Reply:
It's a good idea! I will fix all errors messages in a separate pull request when I will have a free time.

I've added a commit to fix messages format for the `datetime` package here.

Reply:
Thank you!